prov/verbs/src/ep_rdm/verbs_av_ep_rdm.c

```diff
@@ -186,7 +191,7 @@ static int fi_ibv_rdm_av_insert(struct fid_av *av_fid, const void *addr,
 				fi_addr[i] = (uintptr_t) (void *) conn;
 				break;
 			case FI_AV_TABLE:
-				fi_addr[i] = av->count + i;
+				fi_addr[i] = av->used;
```

Comment:
With the logic of the pull request as is, I am seeing that the insertions into an empty table are happening at index 1, instead of index 0. I think that the change should be to use the post-fix increment and the line below it should be swapped. So something like:

av->domain->rdm_cm->conn_table[av->used] = conn;
fi_addr[i] = av->used++;